src/Groups/group_constructors.jl

```diff
@@ -107,24 +127,28 @@ function abelian_group(v::Vector{Int})
 end
 =#
 
-"""
+@doc Markdown.doc"""
     abelian_group(::Type{T}, v::Vector{Int}) where T <: Group -> PcGroup
 
-Return the direct product of cyclic groups of order v[1] x v[2] x ... x v[n], as group of type `T`. Here, `T` must be of type `PermGroup`, `FPGroup` or `PcGroup`.
+Return the direct product of cyclic groups of the orders
+`v[1]`, `v[2]`, $\ldots$, `v[n]`, as an instance of `T`.
+Here, `T` must be one of `PermGroup`, `FPGroup`, or `PcGroup`.
+
+!!! warning
+    The type need to be specified in the input of the function `abelian_group`, otherwise a group of type `GrpAbFinGen` is returned, which is not a GAP group type. In future versions of Oscar, this may change.
```

Comment:
Line wrap?

Also, of course we need to properly integrate the abelian groups provided by Hecke.... but that's another PR

Reply:
Yes.